_data/compiler-options.yml

```diff
@@ -382,13 +432,6 @@
       arg: "path"
       default:
     description: "Class for manifest's Main-Class entry (only useful with -d jar)"
-  - option: "-Xmax-classfile-name"
-    schema:
-      type: "Int"
-      default: "255"
-      min: "72"
-      max: "255"
-    description: "Maximum filename length for generated classes"
```

Comment:
I think this was removed without replacement (fixed at 255). Maybe it would be nice to document that somehow.